integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -5159,6 +5177,36 @@ msgstr "Bild/PNG"
 msgid "Table Document/CSV"
 msgstr "Tabellendokument/CSV"
 
+#: cms/templates/tutorials/page_tree.html:9
+msgid "Introduction - Page Tree"
+msgstr "Einführung in den neuen Seiten-Baum"
+
+#: cms/templates/tutorials/page_tree.html:13
+msgid ""
+"In our new tree view, we have merged the two previous views into one which "
+"includes all previous functions. In this view you are provided with the "
+"following functionalities:"
+msgstr ""
+"In unser neuen Baumansicht haben wir zwei bisherigen Seiten-Ansichten in "
```

Comment:
`Wir haben "Alle Seiten" und "Baumansicht" in eine Ansicht zusammengeführt. In dieser Ansicht können folgende Aktionen durchgeführt werden:`